pkg/client/supplier/client_integration_test.go

```diff
@@ -13,7 +13,8 @@ import (
 )
 
 func TestNewSupplierClient_Localnet(t *testing.T) {
-	t.Skip("TODO_TECHDEBT: this test depends on some setup which is currently not implemented in this test: staked application and servicer with matching services")
+	t.Skip("TODO_TECHDEBT: this test depends on some setup which is currently" +
```

Comment:
Why do we go with `+` for multiline strings instead of backticks? `

https://stackoverflow.com/questions/7933460/how-do-you-write-multiline-strings-in-go